### PR TITLE
Added logging for Dictionary

### PR DESCRIPTION
**Pull request by MarcoStatespace**

### CODE REVIEW

**Review by PizzaStatespace — APPROVED** [review]

File: Runtime/SimpleGraphQL/HttpUtils.cs [hunk 1]
```diff
@@ -441,5 +441,17 @@ await _webSocket.SendAsync(
...
+        {
+            string print = string.Empty;
+            foreach (var pair in dict)
+            {
+                print += pair.Key.ToString() + " - " + pair.Value.ToString();
+                print += "\n";
+            }
+
+            return print;
+        }
```
Comment (FabioStatespace) [inline Runtime/SimpleGraphQL/HttpUtils.cs:455]: We'll use it quite a lot, the more efficient the better
```suggestion
        public static string PrintDictionary<TKey, TValue>(this Dictionary<TKey, TValue> dict)
        {
            StringBuilder print = new StringBuilder();
            foreach (var pair in dict)
            {
                print.AppendFormat("{0} - {1}\n", pair.Key,pair.Value);
            }

            return print.ToString();
        }
```

**Review by MaurizioK — APPROVED** [review]

**Review by MaurizioK — COMMENTED** [review]
@MarcoStatespace you added it on Aimlab Common as well.

Would you consider make this package dependent on AimLab.Common to have just one single function in the entire project?

**Comment by MarcoStatespace** [comment]
@MaurizioK I didn't want to add a dependency tbh. Since it's already spaghetti enough. But I thought we might need it also in the rest of the codebase. 
If that's ok.

**Comment by MarcoStatespace** [comment]
Updated based on Maurizio's suggestion

**Pull request merged**